.secrets.env.example

```diff
@@ -46,3 +46,7 @@ GAIAX_DEPLOYER_PRIVATE_KEY=<random wallet used to deploy>
 
 GAIAX_TESTNET_DEVELOPER_ADDRESS=<address in azure vault membrane>
 GAIAX_API_KEY=<random digit for now>
+
+ARBITRUMSCAN_APIKEY=<Api key arbiscan> 
+ARBI_GOERLI_DEPLOYER_PRIVATE_KEY=<random wallet is used>
+ARBI_GOERLI_DEVELOPER_ADDRESS=<address in azure vault membrane> 
```

Comment:
We should not refer to Azure / Membrane in public repos - just use generic language like "address to be used" or something like that. In this PR we could also change `GAIAX_TESTNET_DEVELOPER_ADDRESS` on line 47 variable to use similar description.

Reply:
Done, generic language is being used.